server/service/src/sync/test/test_data/rnr_form_line.rs

```diff
@@ -50,7 +50,6 @@ fn rnr_form_line_1() -> RnRFormLineRow {
         requested_quantity: 0.0,
         comment: None,
         confirmed: false,
-        previous_average_monthly_consumption: 0.0,
```

Comment:
Weird I swear I did remove this 🤔 Maybe played too dirty with my commit munging

Reply:
You did... it got duplicated XD